pyproject.toml

```diff
@@ -12,22 +12,17 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.9.0,<3.11"
+python = ">=3.10.0,<3.11"
 numpy = "^1.24"
 pandas = "^2.0"
 flwr = "1.9.0"
 opacus = "^1.3.0"
-torch = "^2.1.1"
+torch = "^2.3.0"
 pycyclops = "^0.2.5"
 dp-accounting = "^0.4.3"
 torchmetrics = "^1.3.0"
 aiohttp = "^3.9.3"
 urllib3 = "^2.2.2"
-setuptools = "69.5.1"
```

Comment:
Is this resolved now?

Reply:
Yeah with the update of torch, this is resolved! in torch 2.2.0 they fully move over to the new api from the deprecated one.

More on this: https://github.com/vllm-project/vllm/issues/4961#issuecomment-2141323698

Reply:
Good to know!